api/placer.py

```diff
@@ -475,3 +477,32 @@ def finalize(self):
             'event': 'result',
             'data': result,
         })
+
+class AnalysisPlacer(Placer):
+
+    def check(self):
+        self.requireMetadata()
+        #validators.validate_data(self.metadata, 'analysys.json', 'input', 'POST', optional=True)
+        self.saved = []
+
+    def process_file_field(self, field, info):
+        self.save_file(field)
```

Comment:
Passing info will update the db and trigger job creation (in the new implementation of `save_file` 😄).
DB operations will be done together in the finalize step and, for this first implementation of analysis, automatic job creation has been disabled to limit the complexity (and impact) of the change. 
But it is something we should definetely discuss in some dedicated issue.